index.js

```diff
@@ -214,6 +216,11 @@ class CsvParser extends Transform {
       const chr = buf[i]
       const nextChr = i + 1 < bufLen ? buf[i + 1] : null
 
+      this._currentRowSize++
```

Comment:
```js
if (this.maxRowBytes !== Number.MAX_SAFE_INTEGER) {
  this._currentRowSize++
  if (this._currentRowSize > this.maxRowSize) {
    return cb(new Error('Row exceeds the maximum size'))
  }
}
```

Reply:
I'd then remove the outer condition here. Otherwise we're creating this magic number on which it suddenly would not work 😅

Reply:
Not sure I follow you. I'd prefer if no operation around this feature were performed if `maxRowBytes` is set to the default, do-nothing value.

Reply:
If you want that, I'd take the default back to `0` to make this equal to infinity. This suggestion here introduces `Number.MAX_SAFE_INTEGER` as a magic number - something special happens if this number is set. It's then e.g. not possible to actually set it to `Number.MAX_SAFE_INTEGER` anymore, but someone wanting it would have to set it to `Number.MAX_SAFE_INTEGER-1` to get a comparable result.

Reply:
Different strokes for different folks. Typically you're beholden to the maintainers preferences on PRs :wink: I'll merge this and rectify it afterwards, so you can maintain your position on that in your commit, and the blame will show I modified it.

Reply:
😂 I know ... I just wanted to hear how you'd justify this choice you're taking there. Anyways - thanks for merging it in 🎉